include/UIChar.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef UICHAR_HPP
+# define UICHAR_HPP
+
+# include <string>
+
+class UIChar {
+
+	public:
+		UIChar(std::string const &);
+		UIChar(UIChar const &) = delete;
+		UIChar(UIChar &&) = delete;
+		UIChar &operator=(UIChar const &) = delete;
+		UIChar &operator=(UIChar &&) = delete;
+		virtual ~UIChar(void) = default;
+		
+		std::string getName(void) const;
+		void setScore(int);
```

Comment:
I just realised: i don't see the use of a getter in this case.
A unique setter (but no getter) would be acceptable too.